src/Workspaces/Core/Portable/Workspace/Solution/Solution.cs

```diff
@@ -1633,6 +1641,11 @@ public SolutionChanges GetChanges(Solution oldSolution)
     /// </summary>
     public IReadOnlyList<AnalyzerReference> AnalyzerReferences => this.SolutionState.AnalyzerReferences;
 
+    /// <summary>
+    /// Fallback analyzer config options by language. The set of languages does not need to match the set of langauges of projects included in the surrent solution snapshot.
```

Comment:
'surrent'

Reply:
i think this comment needs to be fleshed out.  UNder what circumstances would these not match?

Reply:
When someone calls WithFallbackAnalyzerOptions directly